sensors/simsensor_common/event_queue.c

```diff
@@ -0,0 +1,118 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Queue for buffering sensor event data from reader to simsensors
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Piotr Nieciecki
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include "event_queue.h"
+
+#include <stdlib.h>
+
+
+struct event_queue_t {
+	sensor_event_t *queue;
+	unsigned int queue_sz;
+	unsigned int act_elem_cnt;
```

Comment:
How about:
```suggestion
	sensor_event_t *queue;
	unsigned int capacity;   /* queue max length */
	unsigned int len;        /* current length of queue */
```
This way it conforms to out coding standard (camelCase) and is easily understandable later in code.